### DISCUSSION TITLE
No entrypoint error though the .dll file is getting created

### DISCUSSION THREAD

**Discussion by AshwinSridharan0410** [paketo-buildpacks/feedback#269] [General]
I am trying to build a dotnet azure functions app of .net6.

It is throwing me an error like this :-
no entrypoint [abc-Function.dll] found: stat /workspace/abc-Function.dll: no such file or directory
This is my .csproj file :-
![image](https://github.com/paketo-buildpacks/feedback/assets/113506379/e2f4159f-496c-4b0f-85e7-e1bc7f8f61cb)
The command that I used:- (I am building it using the dotnet-core buildpack)
pack build azure-functions --buildpack dotnet-core --path=.
![image](https://github.com/paketo-buildpacks/feedback/assets/113506379/7b2a4b6b-7c63-4d3d-8d52-22453f4176e9)
![image](https://github.com/paketo-buildpacks/feedback/assets/113506379/89033e50-546f-43fd-bc8e-c0b3e1f34965)
![image](https://github.com/paketo-buildpacks/feedback/assets/113506379/9edbd50b-6278-4519-b547-2c226a8136d3)
These are the logs for dotnet azure functions.

**Comment by sophiewigmore** [comment]
closing as a duplicate of https://github.com/orgs/paketo-buildpacks/discussions/266
